README.md

```diff
@@ -25,3 +27,13 @@ $ yarn plugin import https://yarnplugins.com/bundle
 ### References
 
 - [Yarn Plugin Tutorial](https://yarnpkg.com/advanced/plugin-tutorial)
+
+### How to join
+
+Open an issue on this repository and ask to be onboarded. Provide relevant information about your plugin like its purpose and sources URL.
+
+Your repo may be merged into this one (like `plugin-bundle`), or may stay separate (like `plugin-deduplicate`). Having your repo on GitHub is preferred but not strictly necessary, discussion can be had.
+
+yarn-plugins.com works by HTTP-redirecting\* an enduser's Yarn from the site's convenient URLs (`https://yarnplugins.com/x`) to the plugin script file hosted within each repo (`https://github.com/user/repo/.../plugin-x.js`). For this reason, you need to be able to permalink your "latest" runnable script. You may be able to achieve this with a build job artifact or release package, but for simpler plugins it might be more proportional to simply commit your artifact next to its sources and use its live repo link.
```

Comment:
The URL to the plugin needs to be starting with `https://raw.githubusercontent.com/...` for GitHub (CDN).